DigitalLearningSolutions.Web.Tests/Controllers/Signposting/ViewResourceControllerTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.Signposting
+{
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.Signposting;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using FakeItEasy;
+    using FluentAssertions.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class ViewResourceControllerTests
+    {
+        private ViewResourceController controller = null!;
+
+        private ILearningResourceReferenceDataService resourceDataService = null!;
+        private ILearningHubAuthApiUrlService urlService = null!;
+        private IUserService userService = null!;
+
+        [SetUp]
+        public void SetUp()
+        {
+            urlService = A.Fake<ILearningHubAuthApiUrlService>();
+            userService = A.Fake<IUserService>();
+            resourceDataService = A.Fake<ILearningResourceReferenceDataService>();
+
+            controller = new ViewResourceController(
+                    userService,
+                    resourceDataService,
+                    urlService
+                ).WithDefaultContext()
+                .WithMockUser(true);
+        }
+
+        [Test]
+        public void ViewResourceController_Index_returns_redirect_to_login_result_if_user_linked()

```

Comment:
Bit of a "nitpik" but given we have an controlled expectation could the test name change slightly from:
`ViewResourceController_Index_returns_redirect_to_login_result_if_user_linked`
to
`ViewResourceController_Index_returns_redirect_to_login_result_when_user_linked`